.ansible-lint

```diff
@@ -4,7 +4,6 @@ exclude_paths:
   - .github/
   - requirements.yml
   - src/**/cookiecutter/{{*
```

Comment:
If the `ansible-lint` is added, CI jobs would fail until ansible-lint are happy with playbooks in all plugins.
But now, only azure plugin fixed some lint issues. We may need to exclude playbooks generated from other plugins. e.g. `test/roles/xxx`.

Reply:
I have added ansible-ignore file instead. I think it's more explicit and less prone to letting new errors slip